components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/config/WebSubAdapterConfiguration.java

```diff
@@ -73,4 +85,24 @@ public String getWebSubHubBaseUrl() {
 
         return webSubHubBaseUrl;
     }
+
+    /**
+     * Returns the configuration value for max number of connections to use in the http connection pool manager.
+     *
+     * @return max number of connections to use in http connection pool manager.
+     */
+    public int getHttpClientMaxConnections() {
+
+        return httpClientMaxConnections;
+    }
+
+    /**
+     * Returns the configuration value for max connections per route to use in the http connection pool manager.
```

Comment:
What does it mean by a 'Route' ?

Reply:
route is the specific remote host that the client is calling